### PR TITLE
chore(RHTAPWATCH-590): remove CloudWatch log forwarding

### PR DESCRIPTION
**Pull request by Omeramsc**
It was agreed to remove application logs from CloudWatch due to its redundancy, as we have the same logs and more in Splunk.

### CODE REVIEW

File: components/monitoring/logging/staging/kustomization.yaml [hunk 1]
```diff
@@ -3,28 +3,3 @@ kind: Kustomization
...
-      kind: ExternalSecret
-  - target:
-      group: external-secrets.io
-      version: v1beta1
-      kind: ExternalSecret
-      name: log-forwarder-splunk-rhtap-application-external-secret
-    patch: |
-      - op: replace
-        path: /spec/dataFrom/0/extract/key
-        value: staging/monitoring/logging/fluentd/splunk-forwarder-rhtap-staging-application
```
Comment (eisraeli) [inline components/monitoring/logging/staging/kustomization.yaml:21]: This patch shouldn't be removed. 
It's for the Splunk-related log forwarders.

File: components/monitoring/logging/staging/kustomization.yaml [hunk 2]
```diff
@@ -3,28 +3,3 @@ kind: Kustomization
...
-  - target:
-      group: external-secrets.io
-      version: v1beta1
-      kind: ExternalSecret
-      name: log-forwarder-splunk-rhtap-audit-external-secret
-    patch: |
-      - op: replace
-        path: /spec/dataFrom/0/extract/key
-        value: staging/monitoring/logging/fluentd/splunk-forwarder-rhtap-staging-audit
```
Comment (eisraeli) [inline components/monitoring/logging/staging/kustomization.yaml:30]: This patch shouldn't be removed. 
It's for the Splunk-related log forwarders.

File: components/monitoring/logging/production/kustomization.yaml [hunk 3]
```diff
@@ -3,28 +3,3 @@ kind: Kustomization
...
-      kind: ExternalSecret
-  - target:
-      group: external-secrets.io
-      version: v1beta1
-      kind: ExternalSecret
-      name: log-forwarder-splunk-rhtap-application-external-secret
-    patch: |
-      - op: replace
-        path: /spec/dataFrom/0/extract/key
-        value: production/monitoring/logging/fluentd/splunk-forwarder-rhtap-production-application
```
Comment (eisraeli) [inline components/monitoring/logging/production/kustomization.yaml:21]: This patch shouldn't be removed. 
It's for the Splunk-related log forwarders.

File: components/monitoring/logging/production/kustomization.yaml [hunk 4]
```diff
@@ -3,28 +3,3 @@ kind: Kustomization
...
-  - target:
-      group: external-secrets.io
-      version: v1beta1
-      kind: ExternalSecret
-      name: log-forwarder-splunk-rhtap-audit-external-secret
-    patch: |
-      - op: replace
-        path: /spec/dataFrom/0/extract/key
-        value: production/monitoring/logging/fluentd/splunk-forwarder-rhtap-production-audit
```
Comment (eisraeli) [inline components/monitoring/logging/production/kustomization.yaml:30]: This patch shouldn't be removed. 
It's for the Splunk-related log forwarders.

File: components/monitoring/logging/production/kustomization.yaml (hunk 3 above)
Comment (Omeramsc) [reply components/monitoring/logging/production/kustomization.yaml]: ack, fixed

File: components/monitoring/logging/production/kustomization.yaml (hunk 4 above)
Comment (Omeramsc) [reply components/monitoring/logging/production/kustomization.yaml]: ack, fixed

File: components/monitoring/logging/staging/kustomization.yaml (hunk 1 above)
Comment (Omeramsc) [reply components/monitoring/logging/staging/kustomization.yaml]: ack, fixed

File: components/monitoring/logging/staging/kustomization.yaml (hunk 2 above)
Comment (Omeramsc) [reply components/monitoring/logging/staging/kustomization.yaml]: ack, fixed

**Pull request merged**